apps/web-mzima-client/src/app/shared/components/collections/collections.component.html

```diff
@@ -77,20 +77,24 @@
   <ng-container *ngIf="!isLoading; else loader">
     <div mat-dialog-content>
       <div class="collection-list">
-        <app-collection-item
-          [collection]="item"
-          [actions]="item.my_collection || isManageCollections"
+        <a
+          *ngFor="let item of collectionList"
+          [routerLink]="['/collections', item.id]"
           class="collection-list__item"
-          (edit)="editCollection(item)"
+          [class.checked]="isPostInCollection(item)"
           (click)="goToCollection(item)"
-          [selectable]="post && !!post.id"
-          (delete)="deleteCollection(item)"
-          *ngFor="let item of collectionList"
-          [checked]="isPostInCollection(item)"
-          (checkedChange)="onCheckChange($event, item)"
           [data-qa]="'collection-item'"
         >
-        </app-collection-item>
+          <app-collection-item
+            [collection]="item"
+            [actions]="item.my_collection || isManageCollections"
+            (edit)="editCollection(item, $event)"
+            (delete)="deleteCollection(item, $event)"
+            [selectable]="post && !!post.id"
+            [checked]="isPostInCollection(item)"
+          >
+          </app-collection-item>
```

Comment:
Still checking. Will let you know by around noon when I come back to it.

Reply:
Okay @Ifycode

Reply:
This solution works @kivuvarosekivuvan Nice work
The focus of the top cards are hidden. You can help fix that.

#

#### Screenshot
<img width="1263" alt="Screenshot 2024-08-09 at 14 46 59" src="https://github.com/user-attachments/assets/c1420721-917e-415a-ab8c-b40562e31d7c">